layouts/partials/hooks/body-end.html

```diff
@@ -7,28 +7,55 @@
 {{/* copy-and-paste helper for codenew shortcode */}}
 {{- if or (.HasShortcode "code_sample") (.HasShortcode "code") (.HasShortcode "codenew") -}}
   {{- if hugo.IsProduction -}}
-    {{- $sweetAlert :=  resources.Get "js/sweetalert-2.1.2.min.js" | minify | fingerprint -}}
-<script async src="{{ $sweetAlert.RelPermalink }}" integrity="{{ $sweetAlert.Data.Integrity }}" crossorigin="anonymous"></script>
+    {{- $toastrJs := resources.Get "js/toastr-2.1.4.min.js" | minify | fingerprint -}}
+    <script async src="{{ $toastrJs.RelPermalink }}"></script>
   {{- else -}}
-    {{- $sweetAlert := resources.Get "js/sweetalert-2.1.2.min.js" -}}
-<script async src="{{ $sweetAlert.RelPermalink }}"></script>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.min.css" rel="stylesheet">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.min.js"></script>
   {{- end -}}
+
 <script type="text/javascript">
-function copyCode(elem){
+function copyCode(elem) {
   if (document.getElementById(elem)) {
     if (navigator.clipboard) {
       navigator.clipboard.writeText(document.getElementById(elem).textContent).then(
         function () {
-          swal("Copied to clipboard: ",elem);
+          toastr.options = {
+            closeButton: true,
+            progressBar: true,
+            positionClass: "toast-bottom-center", 
+            timeOut: 2000, 
+            preventDuplicates: true,   
+            newestOnTop: true          
+          };
+          toastr.success("Copied to clipboard: " + elem, "Success");
```

Comment:
We should localize this string. See https://gohugo.io/content-management/multilingual/